There is a test failure (`//tensorflow/python/kernel_tests:broadcast_to_ops_test`), can you check please?

```
======================================================================
ERROR: testBroadcastToInvalidShape (__main__.BroadcastToTest)
BroadcastToTest.testBroadcastToInvalidShape
----------------------------------------------------------------------
Traceback (most recent call last):
  File ".../tensorflow/python/framework/ops.py", line 1839, in _create_c_op
    c_op = pywrap_tf_session.TF_FinishOperation(op_desc)
google3.third_party.tensorflow.python.framework.errors_impl.InvalidArgumentError: Dimensions must be equal, but are 3 and 2 for '{{node BroadcastTo}} = BroadcastTo[T=DT_INT32, Tidx=DT_INT32](Const, BroadcastTo/shape)' with input shapes: [3], [12] and with input tensors computed as partial shapes: input[1] = [110,53,104,147,157,123,5,24,188,40,5,2].

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File ".../tensorflow/python/kernel_tests/broadcast_to_ops_test.py", line 200, in testBroadcastToInvalidShape
    v = array_ops.broadcast_to(constant_op.constant(x), output_shape)
  File ".../tensorflow/python/ops/gen_array_ops.py", line 855, in broadcast_to
    "BroadcastTo", input=input, shape=shape, name=name)
  File ".../tensorflow/python/framework/op_def_library.py", line 776, in _apply_op_helper
    attrs=attr_protos, op_def=op_def)
  File ".../tensorflow/python/framework/ops.py", line 3518, in _create_op_internal
    op_def=op_def)
  File ".../tensorflow/python/framework/ops.py", line 2002, in __init__
    control_input_ops, op_def)
  File ".../tensorflow/python/framework/ops.py", line 1842, in _create_c_op
    raise ValueError(str(e))
ValueError: Dimensions must be equal, but are 3 and 2 for '{{node BroadcastTo}} = BroadcastTo[T=DT_INT32, Tidx=DT_INT32](Const, BroadcastTo/shape)' with input shapes: [3], [12] and with input tensors computed as partial shapes: input[1] = [110,53,104,147,157,123,5,24,188,40,5,2].
```

--- a/tensorflow/core/framework/tensor_shape.cc
+++ b/tensorflow/core/framework/tensor_shape.cc
@@ -593,7 +593,7 @@ Status MakeShapeHelper(const T* dims, int64 n, Shape* out) {
       if (TF_PREDICT_FALSE(new_num_elements < 0)) {
         TensorShapeProto proto;
         for (int64 j = 0; j < n; ++j) {
-          proto.add_dim()->set_size(dim);
+          proto.add_dim()->set_size(internal::SubtleMustCopy(dims[j]));
         }
         return errors::InvalidArgument(
             "Shape ", TensorShape::DebugString(proto),

--- a/tensorflow/python/kernel_tests/broadcast_to_ops_test.py
+++ b/tensorflow/python/kernel_tests/broadcast_to_ops_test.py
@@ -192,5 +192,15 @@ class BroadcastToTest(test_util.TensorFlowTestCase):
                                                     out, out.get_shape())
     self.assertLess(err, 1e-4)
 
+  def testBroadcastToInvalidShape(self):
+    with self.assertRaisesRegex(
+        (ValueError, errors.InvalidArgumentError),
+        "110,53,104,147,157,123,5,24,188,40,5,2"):
+      output_shape = [110, 53, 104, 147, 157, 123, 5, 24, 188, 40, 5, 2]
+      x = np.array([1, 2, 3], dtype=np.int32)
+      v = array_ops.broadcast_to(constant_op.constant(x), output_shape)
+      self.evaluate(v)
+
+
 if __name__ == "__main__":
   test_lib.main()

